media-proxy/include/session-mtl.h

```diff
@@ -60,7 +60,7 @@ class RxSt20MtlSession : public MtlSession
     size_t source_begin_iova_map_sz;
 #endif
 
-    void copy_connection_params(const mcm_conn_param &request, std::string dev_port);
+    void copy_connection_params(const mcm_conn_param &request, std::string &dev_port);
```

Comment:
C++ reference style: the ampersand sticks to the argument type definition. 
```suggestion
    void copy_connection_params(const mcm_conn_param& request, std::string& dev_port);
```

Consider updating that in all your work.

Reply:
Then .clang-format is not configured correctly.
Can we postpone it for later? Fix .clang-format and apply format globally?

Reply:
Of course, I have no objections.